RELEASE_NOTES.md

```diff
@@ -4,6 +4,7 @@
 - [KOGITO-2905](https://issues.redhat.com/browse/KOGITO-2905) - Change in image definition in the CRDs
 - [KOGITO-3039](https://issues.redhat.com/browse/KOGITO-3039) - Agnostic infrastructure layer configuration on Operator
 - [KOGITO-2113](https://issues.redhat.com/browse/KOGITO-2113) - Simplify binary build deployment
+- [KOGITO-3094](https://issues.redhat.com/browse/KOGITO-3094) - Enable persistence selection in Data Index controller
```

Comment:
You might delete the rest =D

Reply:
do you meant, delete the rest and start a new one?

Reply:
Yes! The "rest" is from the older release. We could have a log of everything, but since this file is used by the CI to create the release notes, I believe we would just make harder for them.

@radtriste once we implement this feature in the CI, the PR could also clean up this file on master.

Reply:
tricky part as some updates could be done on master while release is done ... or we should delete the content on master (as a PR) when creating the release branch. wdyt ?

Reply:
> or we should delete the content on master (as a PR) when creating the release branch.

+1 to that! :D

Reply:
just thinking it would make the process more complicated ... Gonna think about another possibility ...
added to https://issues.redhat.com/browse/KOGITO-2861 anyway ;)